doc/quickstart/intro.rst

```diff
@@ -0,0 +1,151 @@
+ .. role:: html(raw)
+   :format: html
+
+.. _pl_intro:
+
+Introduction
+============
+
+This section is an introduction to how the concept of a :ref:`variational quantum circuit <varcirc>`
+is implemented in PennyLane.
+
+It shows new PennyLane users how to:
+
+.. image:: ../_static/intro_qnodes.png
+    :align: right
+    :width: 180px
+    :target: javascript:void(0);
+
+- Construct quantum circuits via **quantum functions**
+- Define **computational devices**
+- Combine quantum functions and devices to **quantum nodes**
+- Conveniently create quantum nodes using the quantum node **decorator**
+- Find out more about **interfaces** to use quantum nodes
+
+More information about PennyLane's code base can be found in the
+:ref:`Code Documentation <library_overview>`.
+An introduction of how to use and optimize quantum nodes in larger hybrid computations
+is given in the section on :ref:`Interfaces <intro_interfaces>`, and tutorials for different levels of users
+are provided in the :ref:`Tutorials <intro_tutorials>` section.
+
+Quantum functions
+-----------------
+
+A quantum circuit is constructed as a special Python function, a *quantum circuit function*, or *quantum function* in short.
+For example:
+
+.. code-block:: python
+
+    import pennylane as qml
+
+    def my_quantum_function(x, y):
+        qml.RZ(x, wires=0)
+        qml.CNOT(wires=[0,1])
+        qml.RY(y, wires=1)
+        return qml.expval(qml.PauliZ(1))
+
+
+Quantum functions are a restricted subset of Python functions, adhering to the following
+constraints:
+
+* The body of the function must consist of only supported PennyLane
```

Comment:
of only Pennylane supported

Reply:
I think it refers to "supported operations" -> "supported PL operations". I'd suggest to leave it?

Reply:
ok :)